johnsonandjohnson/Bodiless-JS
# feat(webpack): Token Shadowing

## Changes
- Provide token shadowing pattern using webpack normal module replacement.

## Test Instructions
- Visit any page on the starter site which uses an H1.  Verify that the H1 has the 'shadowed-by' attribute.  Verify that this is also the case inside RTE.
- Vist styleguide/editors - verify that rich text editors hve the 'shadowed-by' attribute.

## Related Issues
Closes #1466
